_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: TODO
+exercises: TODO
+questions:
+- "TODO"
+objectives:
+- "Profile the timing of your raster calculations."
+- "Open raster data as a chunked array."
+- "Recognize good practices in selecting proper chunk sizes."
+- "Setup raster calculations that take advantage of parallelization."
+keypoints:
+- "TODO"
```

Comment:
Here's a start:

> * "chunks" are the unit of parallelization
> * Specifying the `chunks` argument with `rioxarray.open_rasterio()` opens a raster as a chunked array
> * the chunks tuple can be adapted to parallelize calculations across different raster dimensions
> * python's `time` module or the jupyter `%%time` magic command can be used to profile calculations

Reply:
Wonderful, thanks! I have included these with some slight changes and added one more.